packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1628,6 +1667,11 @@ export class ContainerRuntime
 	) {
 		this.addMetadataToSummary(summaryTree);
 
+		if (this.runtimeOptions.enableRuntimeIdCompressor && this.idCompressor !== undefined) {
+			const idCompressorState = JSON.stringify(this.idCompressor.serialize(false));
+			addBlobToSummary(summaryTree, idCompressorBlobName, idCompressorState);
```

Comment:
You have a question in the description about whether this should be a blob or a tree. A blob should be fine it the content is small. If the content can be big, you may want to break it down into multiple blobs under a tree for 2 reasons:
1. Keeping blobs small is preferable because they can be uploaded / downloaded quicker.
2. You can do incremental summaries where if some blobs did not change since last summary, you don't need to add the blobs to the summary again.